CSharp/BatchExplorer/readme.txt

```diff
@@ -0,0 +1,11 @@
+﻿Once the app is running, you need to configure it with data from your Azure subscription.
```

Comment:
Can you change this to: "Once BatchExplorer is running, you need to configure it with data about your Azure Batch account.